Add Helm integration (#11531)
DataDog/integrations-core: helm/README.md

# Agent Check: Helm

## Overview

This check monitors Helm deployments through the Datadog Agent.

> Helm supports multiple storage backends. The default in v3 is Kubernetes
> secrets, in v2 it is configmaps. Both are supported by this check.

Reviewer: ```suggestion
Helm supports multiple storage backends. In v3, Helm defaults to Kubernetes secrets and in v2, Helm defaults to ConfigMaps. This check supports both options.
```

## Setup

### Installation

The Helm check is included in the [Datadog Agent][1] package.
No additional installation is needed on your server.

### Configuration

> This is a cluster check. See the [cluster checks documentation][2].

Reviewer: ```suggestion
This is a cluster check. For more information, see the [Cluster checks documentation][2].
```

### Validation

[Run the Agent's status subcommand][3] and look for `helm` under the Checks section.

## Data Collected

### Metrics

This check reports a gauge, `helm.release`, set to 1 for each release deployed

> in the cluster. The metric has tags that identify the Helm release (name, app
> version, chart version, revision, etc.).

Reviewer: ```suggestion
in the cluster. The metric has tags that identify the Helm release such as name, app
version, chart version, and revision.
```
